src/modules/samples/rust/src/sample.rs

```diff
@@ -0,0 +1,2 @@
+// Copyright (c) Microsoft Corporation. All rights reserved..
```

Comment:
empty file intended?

Reply:
Yes, will be used later, it's part of the initial sample rust template